pkg/controller/kedacontroller/kedacontroller_controller.go

```diff
@@ -302,6 +306,11 @@ func (r *ReconcileKedaController) installMetricsServer(instance *kedav1alpha1.Ke
 		transform.EnsureCertInjectionForService(metricsServcerServiceName, injectservingCertAnnotation, injectservingCertAnnotationValue, r.scheme, log),
 		transform.EnsureCertInjectionForDeployment(metricsServerConfigMapName, metricsServcerServiceName, r.scheme, log),
 	}
+	
+	argsPrefixes := []transform.Prefix{transform.ClientCAFile, transform.TLSCertFile, transform.TLSPrivateKeyFile}
```

Comment:
I'd move `argsPrefixes` and `newArgs` definition somewhere around line 302, before `transforms := []mf.Transformer{`.

Then move `transforms = append(transforms, transform.AddPathsToCert...` directly inside the `transforms` defintion.

To kepp it the code consistent